package.json

```diff
@@ -13,6 +13,8 @@
     "release:dry-run": "semantic-release --dry-run",
     "start": "NODE_ENV=development nodemon scripts/start.js --exec babel-node",
     "test": "jest",
+    "test:integration": "jest \"^(?=.*integration.test.js).*\" --forceExit",
```

Comment:
`--forceExit` was required due to the jest tests not exiting. Tried debugging with `--detectOpenHandles` with no luck.

Reply:
I don't like that we're using `--forceExit` because I feel like it's masking an actual issue

Reply:
I think the issue might be we're not unsubscribing from `getRoom`, so the observable doesn't complete and the socket is still opened.

Reply:
I've added the unsubscribe, but it still doesn't exit gracefully. The problem is definitely in the room subscription because it will exit fine after creating and destroying test users. 

@lalli-flores if this is truly a blocker for you, please help investigate.

Reply:
I think we fixed this now? 🤞🏻